src/client/src/containers/AzureFunctionsModal/index.tsx

```diff
@@ -310,17 +304,18 @@ const AzureFunctionsResourceModal = (props: Props) => {
         })}
       >
         <div className={styles.selectionHeaderContainer}>
-          <div>{leftHeader}</div>
+          <div className={styles.leftHeader}>{leftHeader}</div>
           {links[formSectionId] && (
             <a
               tabIndex={disabled ? -1 : 0}
               className={styles.link}
               href={links[formSectionId]}
             >
-              {props.intl.formatMessage(messages.createNew)}
+              {rightHeader}
```

Comment:
This was a bug with the original code, `rightHeader` wasn't being used at all but it's supposed to be here I think

Reply:
Nice catch